src/Console/Mappings/Traits/UpdatesAlias.php

```diff
@@ -47,7 +47,7 @@ protected function getActiveIndex(string $alias):string
      */
     protected function getAlias(string $mapping):string
     {
-        return preg_replace('/^\d{4}\_\d{2}\_\d{2}\_\d{6}\_/', '', $mapping, 1);
+        return preg_replace('/^\d{4}\_\d{2}\_\d{2}\_\d{6}\_(update_)?/', '', $mapping, 1);
```

Comment:
Worth adding a comment so it's clear what this is doing - eg 
> Change `2018_08_16_140102_mapping_name` to `mapping_name`

Reply:
I've made this change to master.